LEAF_Request_Portal/admin/templates/view_history_paginated_html.tpl

```diff
@@ -0,0 +1,6 @@
+<div id="history-slice"></div>
+

```

Comment:
Does the pagination need to be stripped from the original template? Can't it just be hidden if data response doesn't trigger pagination?

Reply:
The html has been separated into another file because of the load order of Vue js modal. The buttons are then hidden is paginated is not applicable.